scripts/prompt_processing_summary.py

```diff
@@ -1,10 +1,13 @@
+import asyncio
 import sys
 import os
 import lsst.daf.butler as dafButler
 from dataclasses import dataclass
 from datetime import date, timedelta
 import requests
 
+from utils import *
```

Comment:
"utils" is a bit of a naming pet-peeve of mine, but I understand that the functions are somewhat varied. Trying to think of some other name options, maybe "db_queries" or "queries"? If nothing else comes to mind then it's ok to just keep it as utils. I'd also list the functions instead of `*`.

Reply:
"queries" is a good name; thanks for the idea. 

Also import specific functions rather than `*` now.